python/test/segy.py

```diff
@@ -558,6 +559,10 @@ def test_read_text_sequence():
             assert text
 
         assert iter(f.text)
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
This is much simple with pytest (I think):

https://docs.pytest.org/en/latest/warnings.html#ensuring-code-triggers-a-deprecation-warning